framework/simd.conf

```diff
@@ -81,34 +81,48 @@ avx512fp16		Leaf07_00EDX	    23	avx512f,f16c	# AVX512 16-bit Floating Point
 amx-tile		Leaf07_00EDX	    24		# Advanced Matrix Extensions Tile support
 amx-int8		Leaf07_00EDX	    25	amx-tile	# AMX Tile multiplication for Int8
 sha512			Leaf07_01EAX	    0	avx	# SHA-512 instructions
-#sm3			Leaf07_01EAX	    1	avx	# SM3 instructions
-#sm4			Leaf07_01EAX	    2	avx	# SM4 instructions
+sm3			Leaf07_01EAX	    1	avx	# SM3 instructions
+sm4			Leaf07_01EAX	    2	avx	# SM4 instructions
 raoint			Leaf07_01EAX	    3		# Remote Atomic Operations, Integer
 avxvnni			Leaf07_01EAX	    4	avx	# AVX (VEX-encoded) versions of the Vector Neural Network Instructions
 avx512bf16		Leaf07_01EAX	    5	avx512f	# AVX512 Brain Float16
+lass			Leaf07_01EAX	    6		# Linear Address Space Seperation
 cmpccxadd		Leaf07_01EAX	    7		# CMPccXADD instructions
 #zlmovsb		Leaf07_01EAX	    10		# Zero-length MOVSB
 #fsrs			Leaf07_01EAX	    11		# Fast Short (REP?) STOSB
 #fsrc			Leaf07_01EAX	    12		# Fast Short (REP?) CMPSB, SCASB
-#fred			Leaf07_01EAX	    17		# Flexible Return and Event Delivery
-#lkgs			Leaf07_01EAX	    18		# Load into Kernel GS
+fred			Leaf07_01EAX	    17		# Flexible Return and Event Delivery
+lkgs			Leaf07_01EAX	    18		# Load into Kernel GS
```

Comment:
Can we make use of these and `lass` in userspace (`lam` makes sense)? If not, leave commented out.

Reply:
Primarily kernel-space, the deeper I dig. I'll leave them commented.